package.json

```diff
@@ -6,8 +6,8 @@
     "test:ci": "jest --clearCache && lerna run test:ci --stream",
     "codecov": "curl -s https://codecov.io/bash | bash -s",
     "e2e:prod:all": "cd e2e/project/ && npx frontity build && npx frontity serve --port 3001",
-    "e2e:prod:module": "cd e2e/project/ && npx frontity build --target module && npx frontity serve --port 3001",
-    "e2e:prod:es5": "cd e2e/project/ && npx frontity build --target es5 && npx frontity serve --port 3001",
+    "e2e:prod:module": "cd e2e/project/ && npx frontity build --publicPath /custom/path --target module && npx frontity serve --port 3001",
+    "e2e:prod:es5": "cd e2e/project/ && npx frontity build --publicPath /custom/path/ --target es5 && npx frontity serve --port 3001",
```

Comment:
I think this is fine as a workaround for now, but in the future, I suggest that we create a `e2e.test.js` node script that we can call from the github e2e-tests action [here](https://github.com/frontity/frontity/blob/dev/.github/workflows/e2e-tests.yml#L34)

And that script could take any number of parameters for anything that we want to test:
- target: `es5`
- target: `module`
- publicPath: `/custom/path`
- no public path specified

and whatever other params we have in the future.